deploy/kessel-inventory-ephem.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: inventory
+objects:
+  - apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: inventory-api-config
+    data:
+      inventory-api-config.yaml: |
+        authn:
+          psk:
+            pre-shared-key-file: /psks/psks.yaml
+        authz:
+          kessel:
```

Comment:
Does it not require to configuring the URL point to relations API?

Reply:
Hey @akoserwal, with the new config package, the relations API URL is configured by parsing the clowder appconfig and setting the values direclty
https://github.com/project-kessel/inventory-api/blob/3e3eb9495a4cd870b2979db73a2a0959c60fc473/internal/config/config.go#L70

Reply:
ack, thanks